go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/segmentio/kafka-go
 go 1.11
 
 require (
-	github.com/DataDog/zstd v1.4.0
 	github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
 	github.com/golang/snappy v0.0.1
+	github.com/google/go-cmp v0.4.0 // indirect
```

Comment:
This seems to be caused by `klauspost/comrpess` not having a `go.mod` file, I opened https://github.com/klauspost/compress/issues/208 to follow up.

Reply:
v1.9.8 is released which removes this dependency. Incidentally it also has faster/better gzip.